umbrella/src/lib.rs

```diff
@@ -882,6 +882,10 @@ pub use polkadot_node_subsystem_util;
 #[cfg(feature = "polkadot-overseer")]
 pub use polkadot_overseer;
 
+/// Runs a polkadot parachain node which could be a collator.
```

Comment:
This is coming from `Cargo.toml` file of the omnoi node, and is quite obsolete. Can you make it a bit better please?

Reply:
Done. Fixed.